src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mds-summary.pipe.ts

```diff
@@ -42,22 +42,49 @@ export class MdsSummaryPipe implements PipeTransform {
       contentLine1 = `${active} ${this.i18n('active')}`;
       contentLine2 = `${standbys + standbyReplay} ${this.i18n('standby')}`;
     }
-
+    let standbyHoverText = '';
+    let standbyflg = false;
+    for (const i in value.standbys) {
+      if (standbyflg) {
+        standbyHoverText = standbyHoverText + ', ' +  value.standbys[i].name;
+      } else {
+        standbyHoverText = standbyHoverText + value.standbys[i].name;
+        standbyflg = true;
+      }
+    }
+    const fsLength = (value.filesystems) ? value.filesystems.length : 0;
+    const infoObject = fsLength > 0 ? value.filesystems[0].mdsmap.info : {};
+    let activeHoverText;
+    Object.keys(infoObject).forEach(key => {
+      activeHoverText = infoObject[key].name;
+    });
```

Comment:
I'm not sure about this what if we would have more info attributes than just one? Shouldn't we present them in a comma separated list too, rather than just using the last value presented through `forEach`?

If we should present a comma separated list - here is my suggestion :)
```suggestion
    const activeHoverText = Object.values(infoObject)
      .map((info: any): string => info.name)
      .join(', ');
```

Reply:
It seems so, I haven't done it myself but there is a [documentation page](https://docs.ceph.com/docs/master/cephfs/multimds/) about exactly that.